generators/generator-constants.js

```diff
@@ -48,8 +48,8 @@ const KAFKA_VERSION = '5.5.2';
 const JACKSON_DATABIND_NULLABLE_VERSION = '0.2.1';
 
 // NPM packages version
-const HUSKY_VERSION = '4.2.5';
-const LINT_STAGED_VERSION = '10.4.2';
+const HUSKY_VERSION = '4.3.0';
```

Comment:
These versions should probably be managed by https://github.com/jhipster/generator-jhipster/blob/main/generators/client/templates/common/package.json now